app/angular/src/server/angular-cli_config.ts

```diff
@@ -67,9 +68,26 @@ export function getLeadingAngularCliProject(ngCliConfig: any) {
     throw new Error('angular.json must have projects entry.');
   }
 
-  const fallbackProject = defaultProject && projects[defaultProject];
-  const firstProject = projects[Object.keys(projects)[0]];
-  return projects.storybook || fallbackProject || firstProject;
+  let projectName;
+  const firstProjectName = Object.keys(projects)[0];
```

Comment:
Is `projects` guaranteed to have at least one key?

Reply:
It must, or else we are speaking of an empty project.
A couple of lines earlier there's also a check for making sure there is at least one project
https://github.com/storybookjs/storybook/blob/f419b7fa21aff07406bb3c8fe219b11ce50c81b9/app/angular/src/server/angular-cli_config.ts#L67